docs/user/tutorials/01-30-register-secured-api.md

```diff
@@ -0,0 +1,307 @@
+  name: {SECRET_NAME}
+  namespace: kyma-system
+data:
+  clientId: {BASE64_ENCODED_CLIENT_ID}
+  clientSecret: {BASE64_ENCODED_CLIENT_SECRET}
+```
+
+To create such a Secret, run this command:
+
+```bash
+kubectl create secret generic {SECRET_NAME} --from-literal clientId={CLIENT_ID} --from-literal clientSecret={CLIENT_SECRET} -n kyma-system
+```
+## Register an OAuth 2.0 mTLS-secured API
+
+This is an example of the **service** object for an API secured with OAuth where the token is fetched from an mTLS-secured endpoint:
+
+```yaml
+  - id: {TARGET_UUID}
+    name: my-mTLS-oauth-service
+    displayName: "My mTLS OAuth Service"    
+    description: "My service"
+    providerDisplayName: "My organisation"
+    entries:
+    - credentials:
+        secretName: {SECRET_NAME}
+        authenticationUrl: {OAUTH_TOKEN_URL}
+        type: OAuthWithCert
+      targetUrl: {TARGET_API_URL}
+      type: API
+```
+
+This is an example of the Secret containing credentials:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {SECRET_NAME}
+  namespace: kyma-system
+data:
+  clientId: {BASE64_ENCODED_CLIENT_ID}
+  crt: {BASE64_ENCODED_CERTIFICATE}
+  key: {BASE64_ENCODED_PRIVATE_KEY}
+```
+
+To create such a Secret, run this command:
+
+```bash
+kubectl create secret generic {SECRET_NAME} --from-literal clientId={CLIENT_ID} --from-literal crt={CERTIFICATE} --from-literal key={PRIVATE_KEY} -n kyma-system
+```
+
+## Register a client certificate-secured API
+
+This is an example of the **service** object for an API secured with a client certificate:
+
+```yaml
+  - id: {TARGET_UUID}
+    name: my-client-cert-service
+    displayName: "My Client Cert Service"
+    description: "My service"
+    providerDisplayName: "My organisation"
+    entries:
+    - credentials:
+        secretName: {SECRET_NAME}
+        type: CertificateGen
+      targetUrl: {TARGET_API_URL}
+      type: API
+```
+
+This is an example of the Secret containing credentials: 
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {SECRET_NAME}
+  namespace: kyma-system
+data:
+  crt: {BASE64_ENCODED_CERTIFICATE}
+  key: {BASE64_ENCODED_PRIVATE_KEY}
+```
+
+To create such a Secret, run this command:
+
+```bash
+kubectl create secret generic {SECRET_NAME} --from-literal crt={CERTIFICATE} --from-literal key={PRIVATE_KEY} -n kyma-system
+```
+
+## Register a CSRF-protected API
+
+This is an example of the **service** object for an API secured with both Basic Authentication and a CSRF token:
+
+```yaml
+  - id: {TARGET_UUID} 
+    name: my-csrf-service
+    displayName: "My CSRF Service" 
+    description: "My service"
+    providerDisplayName: "My organisation"
+    entries:
+    - credentials:
+        secretName: {SECRET_NAME}
+        type: Basic
+        csrfInfo:
+          tokenEndpointURL: {CSRF_TOKEN_URL}
+      targetUrl: {TARGET_API_URL}
+      type: API
+```
+
+> **NOTE:** It is assumed that the CSRF token endpoint service uses the same credentials as the target API.
```

Comment:
```suggestion
> **NOTE:** The example assumes that the CSRF token endpoint service uses the same credentials as the target API.
```
Do we assume that in the example or in general? I'd try to avoid the passive voice if possible